code/api/src/config/database.json

```diff
@@ -7,6 +7,14 @@
     "dialect": "postgres",
     "seederStorage": "sequelize"
   },
+  "test": {
+    "username": "philipdefraties",
```

Comment:
We should change this to `'postgres'` so that it doesn't break on other machines.

Reply:
Thanks for catching that, I made that when I was following along with the testing lesson

Reply:
No problem! It should be updated now.